ClientApp/src/Pages/SearchSightings/SearchSightings.scss

```diff
@@ -2,26 +2,49 @@
 @import '../../Styles/styles';
 
 
+
 .search-sighting-page {
-    margin: $gutter-width;
-    height: 100%;
-  }
+  margin: $gutter-width;
+  height: 100%;
+}
   
-  .search-sighting-form {
-    position: relative;
-    left: 50%;
-    transform: translateX(-50%);
+.search-sighting-form {
+  position: relative;
+  left: 50%;
+  transform: translateX(-50%);
     
   
-    .form-label {
-      @include formLabel;
-    }
+  .form-label {
+    @include formLabel;
+  }
   
-    .form-input {
-      @include formInput;
-    }
+  .form-input {
+    @include formInput;
+  }
   
-    .submit-button {
-      @include primaryButton;
-    }
-  }
\ No newline at end of file
+  .submit-button {
+     @include primaryButton;
+  }
+}
+
+
+tr:nth-child(even){
+
+  background-color: $accent-color
+}
+
+td,th {
+  padding-left: 7.5px;
```

Comment:
I'm fine to merge this but this will actually get rounded to 8px I think as decimals aren't respected in pixels, they represent the smallest unit

Reply:
Cool, will change this is my paginated branch!